tests/bach_tests.ts

```diff
@@ -24,7 +25,7 @@ function minuet1(options: TestOptions): void {
   const id = (id: string) => registry.getElementById(id) as StaveNote;
 
   const f: Factory = VexFlowTests.makeFactory(options, 1100, 900);
-  const score = f.EasyScore({ throwOnError: true });
+  const score = f.EasyScore({ throwOnError: true } as EasyScoreOptions);
```

Comment:
EasyScoreOptions should allow optional fields to avoid this casting.

Reply:
Yup. The final product looks like: https://github.com/ronyeh/vexflow/blob/3118234a440b0549e6d89fea09a145861a70a621/src/easyscore.ts#L389-L394

If you'd like me to include all those changes in this PR, let me know. It will make this PR bigger, and it might have to include some of the font fixes.

Reply:
we can leave this one cast and resolve it in the next PR